feat(trace-view): Add documentation for the new Trace View (#3347)
getsentry/sentry-docs: src/docs/product/issues/issue-details/index.mdx

In addition, this page provides several other key pieces of information.

## Quick Trace

> The quick trace (below the date) shows an abbreviated view of the related [trace](/product/performance/distributed-tracing) for the current transaction.

Reviewer: ```suggestion
The quick trace (which displays below the date) is an abbreviated view of the related [trace](/product/performance/distributed-tracing) for the current transaction.
```

> The quick trace will show up to 6 nodes representing the different groups of the event's trace.

Reviewer: ```suggestion
The quick trace displays up to six nodes, each representing different groups of the event's trace:
```

> - **Root**: The transaction that started the trace
> - **Ancestor**: The events that are descendants of the root, and are directly related to the current event
> - **Parent**: The event that directly started the current event
> - **This Event**: A node representing the current event that you are viewing
> - **Children**: Events that the current event directly started
> - **Descendants**: Any future generations of events that children of this event create

Reviewer: ```suggestion
- **Root**: Starting transaction
- **Ancestor**: Events that are descendants of the root, and are directly related to the current event
- **Parent**: Event that directly started the current event
- **This Event**: Node representing the current event being viewed
- **Children**: Events that the current event directly started
- **Descendants**: Future generations of events created by children of this event
```

> A node will turn red if there are any errors associated with events in that group.

Reviewer: ```suggestion
A node will be red if it is associated with errors.
```

> Clicking View Full Trace will bring you to the [Trace View](/product/performance/trace-view)

Reviewer: ```suggestion
Click "View Full Trace" to display the [Trace View](/product/performance/trace-view).
```
